js/lang/en.js

```diff
@@ -39,6 +39,7 @@ app.lang['en'] = (function() {
 		'payment-details.date': 'Date',
 		'payment-details.amount': 'Amount',
 		'payment-request.data.must-be-object': '"data" must be an object.',
+		'sample-addresses.label': 'Sample Addresses:',
```

Comment:
Don't you want to add also the same label for the other languages?

Reply:
If a language string is missing, the default language is used (en) instead. So in this case, if the display language is Czech, the english version of this string will be displayed instead.